PyInstaller/hooks/hook-PySide.QtGui.py

```diff
@@ -0,0 +1,24 @@
+#-----------------------------------------------------------------------------
+# Copyright (c) 2013, PyInstaller Development Team.
+#
+# Distributed under the terms of the GNU General Public License with exception
+# for distributing bootloader.
+#
+# The full license is in the file COPYING.txt, distributed with this software.
+#-----------------------------------------------------------------------------
+
+
+hiddenimports = ['PySide.QtCore']
+
+from PyInstaller.hooks.hookutils import qt4_plugins_binaries
+
+ns = "PySide"
+
+def hook(mod):
+    # TODO fix this hook to use attribute 'binaries'.
+    mod.pyinstaller_binaries.extend(qt4_plugins_binaries('accessible', ns=ns))
```

Comment:
hmm I don't quite understand... and then what should the function return?

looking at https://github.com/pyinstaller/pyinstaller/blob/python3/PyInstaller/hooks/hook-PyQt4.QtGui.py#L17 I understand now that the TODO was referring to call mod.binaries.add instead of extending the specific pyinstaller_binaries attribute, but I might be wrong.